src/Tribe/JSON_LD/Order.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+/**
+ * A JSON-LD class to hook and change how Orders work on Events
+ * @todo rework this class to make it standalone from The Events Calendar
+ */
+class Tribe__Tickets__JSON_LD__Order {
+
+	/**
+	 * Get (and instantiate, if necessary) the instance of the class
+	 *
+	 * @static
+	 * @return self
+	 *
+	 */
+	public static function instance() {
+		static $instance;
+
+		if ( ! $instance instanceof self ) {
+			$instance = new self;
+		}
+
+		return $instance;
+	}
+
+	/**
+	 * Value indicating low stock availability for a specific ticket.
+	 *
+	 * This can be overridden with the tribe_tickets_json_ld_low_inventory_level filter.
+	 *
+	 * @var int
+	 */
+	protected $low_stock = 5;
+
+	/**
+	 * Setup Google Event Data for tickets.
+	 */
+	public static function hook() {
```

Comment:
Mostly asking to satisfy my curiosity: but we have `hook()` and `instance()` static methods as well as a public constructor. 

* If it's a singleton, should we make the constructor protected/private? 
* Should the hooks we set up inside the constructor be moved to the `hook()` method?
* Do we _need_ `hook()` *and* `instance()` (or, is there a reason for both to be public)?